tripal/tests/src/Functional/TripalRoutePermissionsTest.php

```diff
@@ -19,7 +19,7 @@ class TripalRoutePermissionsTest extends BrowserTestBase {
   // protected $htmlOutputEnabled = TRUE;
   protected $defaultTheme = 'stark';
 
-  protected static $modules = ['tripal', 'file', 'field_ui'];
+  protected static $modules = ['tripal', 'tripal_chado', 'file', 'field_ui'];
```

Comment:
This shows a circular dependency 🤔 Which routes specifically needed this added? Instead of adding tripal_chado here we should create a TripalChadoRoutePermissionsTest in the tripal_chado module that tests those specific routes and then remove them from here.

Reply:
I see in the PR description that the following were all causing inaccessibility issues: loaders, storage, files. Did none of these work without tripal_chado added?

Reply:
Correct, without tripal_chado all of those failed.